server/models/postgis/user.py

```diff
@@ -0,0 +1,66 @@
+#import geojson
+from server import db
+#from server.models.dtos.user_dto import UserDTO, UserFilterDTO, Pagination, UserSearchQuery, UserSearchDTO, \
+#     ListedUser
+from server.models.dtos.user_dto import UserDTO
+from server.models.postgis.lookups import UserRole
+from server.models.postgis.utils import NotFound, timestamp
+
+
+class User(db.Model):
+    """ Describes the user account table """
+    __tablename__ = "dmis_users"
```

Comment:
I think I'd rather keep the name like this for now to distinguish the app control tables from spatial data tables.  Will rename if I get it using a different schema